platform/darwin/src/MGLOfflineStorage.h

```diff
@@ -195,6 +195,13 @@ typedef void (^MGLOfflinePackRemovalCompletionHandler)(NSError * _Nullable error
  KVO change notifications on the shared offline storage object’s `packs` key
  path. Removals from that array result in an `NSKeyValueChangeRemoval` change.
  
+ @note When you remove an offline pack, any resources that are required by that
+    pack, but not other packs, become eligible for removal from the offline
+    database. Because the offline database is also used as a general purpose
```

Comment:
Using “offline storage” here adds confusion, because a database is a storage medium, whereas “storage” is vague and could be interpreted to refer to more abstract elements of MGLOfflineStorage.

>  When you remove an offline pack, any resources that are required by that pack, but not other packs, become eligible for _deletion from offline storage_. Because _offline storage_ is also used as a general purpose cache for map resources, such resources may not be immediately removed if the implementation determines that they remain useful for general performance of the map.

It strikes me as strange to say, "Because offline storage is also used as a general purpose cache for map resources”...